sharktank/sharktank/layers/testing.py

```diff
@@ -51,80 +51,139 @@ def make_llama_attention_block_theta(
-                data=make_rand_torch((18432,), dtype=dtype)
+                data=make_rand_torch((mlp_hidden_size3,), dtype=dtype)
             ),
             "img_mod.lin.weight": DefaultPrimitiveTensor(
-                data=make_rand_torch((18432, 3072), dtype=dtype)
+                data=make_rand_torch((mlp_hidden_size3, hidden_size), dtype=dtype)
             ),
             "txt_attn.norm.key_norm.weight": DefaultPrimitiveTensor(  #
-                data=make_rand_torch((128,), dtype=dtype)
+                data=make_rand_torch((in_channels,), dtype=dtype)
             ),
             "txt_attn.norm.query_norm.weight": DefaultPrimitiveTensor(  #
-                data=make_rand_torch((128,), dtype=dtype)
+                data=make_rand_torch((in_channels,), dtype=dtype)
             ),
             "txt_attn.proj.bias": DefaultPrimitiveTensor(
-                data=make_rand_torch((3072,), dtype=dtype)
+                data=make_rand_torch((hidden_size,), dtype=dtype)
             ),
             "txt_attn.proj.weight": DefaultPrimitiveTensor(
-                data=make_rand_torch((3072, 3072), dtype=dtype)
+                data=make_rand_torch((hidden_size, hidden_size), dtype=dtype)
             ),
             "txt_attn.qkv.bias": DefaultPrimitiveTensor(
-                data=make_rand_torch((9216,), dtype=dtype)
+                data=make_rand_torch((mlp_hidden_size,), dtype=dtype)
             ),
             "txt_attn.qkv.weight": DefaultPrimitiveTensor(
-                data=make_rand_torch((9216, 3072), dtype=dtype)
+                data=make_rand_torch((mlp_hidden_size, hidden_size), dtype=dtype)
             ),
             "txt_mlp.0.bias": DefaultPrimitiveTensor(
-                data=make_rand_torch((12288), dtype=dtype)
+                data=make_rand_torch((mlp_hidden_size2), dtype=dtype)
             ),
             "txt_mlp.0.weight": DefaultPrimitiveTensor(
-                data=make_rand_torch((12288, 3072), dtype=dtype)
+                data=make_rand_torch((mlp_hidden_size2, hidden_size), dtype=dtype)
             ),
             "txt_mlp.2.bias": DefaultPrimitiveTensor(
-                data=make_rand_torch((3072), dtype=dtype)
+                data=make_rand_torch((hidden_size), dtype=dtype)
             ),
             "txt_mlp.2.weight": DefaultPrimitiveTensor(
-                data=make_rand_torch((3072, 12288), dtype=dtype)
+                data=make_rand_torch((hidden_size, mlp_hidden_size2), dtype=dtype)
             ),
             "txt_mod.lin.bias": DefaultPrimitiveTensor(
-                data=make_rand_torch((18432,), dtype=dtype)
+                data=make_rand_torch((mlp_hidden_size3,), dtype=dtype)
             ),
             "txt_mod.lin.weight": DefaultPrimitiveTensor(
-                data=make_rand_torch((18432, 3072), dtype=dtype)
+                data=make_rand_torch((mlp_hidden_size3, hidden_size), dtype=dtype)
+            ),
+        }
+    )
+
+
+def make_mmdit_single_block_theta(
```

Comment:
The naming would benefit if it clarified that it creates a random theta.

Reply:
Done.